packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.ts

```diff
@@ -274,6 +275,59 @@ export function makeMongoDbEmbeddedContentStore({
       }
     },
 
+    async listDataSources(): Promise<DataSourceMetadata[]> {
```

Comment:
pls add some basic tests for this method to `MongoDbEmbeddedContentStore.test.ts`. can use `mongodb-memory-server` for these tests to keep simple.